.github/workflows/tf-apply-hash.yml

```
@@ -0,0 +1,129 @@
+name: HASH Terraform Apply
+
+# TODO: While this workflow is specific to HASH, we should consider
+#  making this a reusable workflow, parameterized on the directory and environment.
+
+on:
+  workflow_dispatch:
+    inputs:
+      env:
+        type: choice
+        options: ["prod"]
+        description: "Environment to deploy"
+        required: true
+        default: "prod"
+
+  push:
+    branches:
+      - main
+    paths:
+      - "infra/internal-api/**"
```

I think these needs adjustment
```suggestion
      - "infra/terraform/hash/**"
      - "infra/terraform/modules/**"
```

---

You're right, thanks for catching!